examples/rbfe_edge_list.py

```diff
@@ -83,11 +82,12 @@ def read_from_args():
             mol_b = get_mol_by_name(mols, mol_b_name)
 
             print(f"Submitting job for {mol_a_name} -> {mol_b_name}")
+            mcs_threshold = 100.0
```

Comment:
This seems really large. 100 angstroms of distance seems to pretty much mean no threshold?

Reply:
this is actually a problem it turns out - (i will patch this a bit)

Reply:
resolved - there was a bug in the MCS logic (dealing with how the threshold is mapped, there was a corner cases where the lowest cost[i,j] may still contain bad pair that was bigger than the distance threshold.